### PR TITLE
feat: [B2BP-420] - editorial store badges

### PR DESCRIPTION
**Pull request by Meolocious**
## Short description
This PR will add the store buttons (Google Play and App Store) for the Editorial component. Those buttons are optional and they're used only if the href is passed throught the storeButtons. Also, the bug about the build that was missing the "images" folder is fixed.

### Preview
If it makes sense, please add one or more screenshots/videos.

## List of changes proposed in this pull request
- Added storeButtons to Ctas.tsx file as optional
- Imported new storeButtons inside the Editorial.tsx file
- Updated "Dark" and "Light" Storybook version
- Fixed package.json to copy the "images" folder inside "dist"

## How to test
Tested in local running the storybook. Then builded and passed the "dist" folder inside the B2BP project.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -88,7 +88,7 @@
     "storybook": "storybook dev -p 6006",
     "build-storybook": "storybook build",
     "test": "jest",
-    "copy:assets": "cpx 'src/assets/**' 'dist/assets'"
+    "copy:assets": "npx cpx \"src/assets/**\" \"dist/assets\""
```
Comment (kin0992) [inline package.json:91]: `cpx` is a `devDependency` so I thing the original command should work. Why did you change it? What issue are you going to fix?

Comment (Meolocious) [reply package.json:91]: Unfortunately, the original command doesn nothing. If you try the original one running "npm run copy:assets" you'll see that the folder "images" is never copied inside the assets folders from "dist" (and this caused us a lot of issues on one of the components on B2BP). After several tries, this is the one that worked 100% of the times and we need the "images" folder for this PR due to the badges we implemented.

I'm not a fan to change scrips, but seemed that no one tested that one and thinked that worked only 'cause 2 folders on 3 of them were copied

Comment (datalek) [reply package.json:91]: Checking the [last version](https://www.npmjs.com/package/@pagopa/pagopa-editorial-components/v/3.1.2?activeTab=code) I see the folder `images` and its content seems right to me:

Screenshot 2024-03-26 at 15 04 37

File: src/components/Editorial/Ctas.tsx [hunk 2]
```diff
@@ -2,22 +2,47 @@ import Button, { type ButtonProps } from '@mui/material/Button';
...
+  breakpoints: {
+    values: {
+      xs: 0,
+      sm: 420,
+      md: 900,
+      lg: 1200,
+      xl: 1536,
+    },
+  },
+});
```
Comment (datalek) [inline src/components/Editorial/Ctas.tsx:25]: What problem does this custom theme solve? Can the caller change the theme even though it is defined this way here?

Comment (Meolocious) [reply src/components/Editorial/Ctas.tsx:25]: The issue about this point is for the "sm" breakdown: to adapt the badges, Salvatore needed to change that break to 420 so he did this part as you see. The other breakdowns are not changed from the regular expression but he founds errors if he didn't put them all inside the values.

Comment (datalek) [reply src/components/Editorial/Ctas.tsx:25]: Can the caller change the theme even though it is defined this way here?
